### ISSUE TITLE
analyse_texts_to_cas returns results only after all data has been processed

### ISSUE THREAD

**Issue by DavidHuebner** [averbis/averbis-python-api#115]
**Describe the bug**
Essentially a duplicate of https://github.com/averbis/averbis-python-api/issues/101, just for the endpoint `analyze_texts_to_cas`. The endpoint returns results only after all data has been processed - but it should already start returning results as the data is being processed.
